### PR TITLE
feat: implement Token Panel on the Dashboard

### LINKED ISSUE #3222: [Demand] Implement Token Panel on the Dashboard
## Catalogue

- [ ] UI/UX
- [ ] Mask Core Feature
- [x] Mask Plugin Feature (ITO)
- [ ] Security/Cryptography Feature
- [ ] Other
  - /_List it here_/

## Part 1: Introduction and Background

Migrate ITO composition dialog onto the token panel of the new Dashboard.

image

### PR DESCRIPTION
**Pull request by UncleBill**
closes #3222

### CODE REVIEW

File: packages/maskbook/src/_locales/en/messages.json [hunk 1]
```diff
@@ -632,5 +632,6 @@
     "wallet_transfer_error_amount_absence": "Enter an amount",
     "wallet_transfer_error_address_absence": "Enter recipient address",
     "wallet_transfer_error_invalid_address": "Invalid recipient address",
-    "wallet_transfer_error_insufficent_balance": "Insufficient {{symbol}} balance"
+    "wallet_transfer_error_insufficent_balance": "Insufficient {{symbol}} balance",
+    "token": "Token"
```
Comment (septs) [inline packages/maskbook/src/_locales/en/messages.json:636]: plz re-order this field position.

Comment (UncleBill) [reply packages/maskbook/src/_locales/en/messages.json]: done

File: packages/maskbook/src/extension/options-page/Route.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ export enum DashboardRoute {
     Settings = '/settings',
     Plugins = '/plugins',
     Setup = '/setup',
+    Token = '/token',
```
Comment (Jack-Works) [inline packages/maskbook/src/extension/options-page/Route.ts:9]: Isn't it a subroute of the wallet?

Comment (UncleBill) [reply packages/maskbook/src/extension/options-page/Route.ts:9]: No. /cc @guanbinrui

**Comment by septs** [comment]
fix resolve conflict
fix build failed

**Comment by UncleBill** [comment]
The feature has been cancelled.

**Pull request closed without merging**